src/app/shared/components/tooltip/tooltip.component.ts

```diff
@@ -1,64 +1,22 @@
-import { ChangeDetectorRef, Component, ElementRef, Input, OnChanges, OnDestroy, SimpleChange, SimpleChanges, ViewChild } from '@angular/core'
-import { TooltipMousePosition } from 'src/app/core/models/tooltips'
+import { Component, ElementRef, HostBinding, TemplateRef } from '@angular/core'
 
 @Component({
   selector: 'sofie-tooltip',
   templateUrl: './tooltip.component.html',
   styleUrls: ['./tooltip.component.scss'],
 })
-export class TooltipComponent implements OnChanges, OnDestroy {
-  @Input() public tooltipElementHoverMousePosition?: TooltipMousePosition
-  private isTooltipAppended: boolean = false
+export class TooltipComponent {
+  @HostBinding('style.top.px')
+  public topPosition: number = 0
```

Comment:
It would be nice with a unit on this and `leftPosition`.

Reply:
Updated